CaloFilters/src/CaloCosmicCalib_module.cc

```diff
@@ -199,17 +210,17 @@ namespace mu2e
 	//====================================
 	// Now fill up the selection flag  ...
 	//====================================
-	if( Eouter>EMIPcut && Einner>EMIPcut && NumMipCel>_mincelinout ) selflag=selflag+1;   // Out-IN or IN-Out
+	if( Eouter>_mincelcut && Einner>_mincelcut && NumMipCel>_mincelinout ) selflag=selflag+1;   // Out-IN or IN-Out
 	
-	if( NumMipCel>_minceldiag && YoutMax>-999. && YoutMin<999. ){	  
+	if( NumMipCel>_minceldiag ){ // 	  
 	  DXmax = abs(XMax-XMin);
 	  DYmax = abs(YoutMax-YoutMin);
 	  float DiagVal=0.;
 	  if( DXmax> 0 ) DiagVal = DYmax/DXmax;	  
-	  if( DXmax  < 35.0  ) selflag = selflag+100;  // Vertical selection
-	  if( DiagVal > 1.5 && DiagVal < 2.5 )
+	  if( DXmax  < _crystalsize  ) selflag = selflag+100;  // Vertical selection, cell size+1mm
```

Comment:
I see that _crystal size now comes from fcl.  Should come from the geometry system or is it some effective number?

Reply:
Thanks.  I see that it's fixed now.